draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -857,8 +886,62 @@ are not blinded, but contain disclosable claims. This limitation
 needs to be taken into account by Issuers when creating the structure of
 the SD-JWT.
 
+## Issuer Signature Key Distribution and Rotation {#issuer_signature_key_distribution}
+
+This specification does not define how signature verification keys of
+Issuers are distributed to Verifiers. However, it is RECOMMENDED that
+Issuers publish their keys in a way that allows for efficient and secure
+key rotation and revocation, for example, by publishing keys at a
+predefined location using the JSON Web Key Set (JWKS) format [@RFC7517].
+Verifiers need to ensure that they are not using expired or revoked keys
+for signature verification as reasonable and appropriate for the given
+key distribution method.
+
 # Privacy Considerations {#privacy_considerations}
 
+## Storage of Signed User Data
+
+Wherever End-User data is stored, it represents a potentially
+interesting target for an attacker. This target can be of particularly
+high value when the data is signed by a trusted authority like an
+official national identity service. For example, in OpenID Connect,
+signed ID Tokens can be stored by Relying Parties. In the case of
+SD-JWT, Holders have to store signed SD-JWTs and associated Disclosures,
+and Issuers and Verifiers may decide to do so as well.
+
+As usual, a leak of such data risks revealing private data of End-Users
```

Comment:
```suggestion
A leak of such data risks revealing private data of End-Users
```

Reply:
I'd like to keep the notion that this sentence is not telling anything new. What about this?

```suggestion
Not surprisingly, a leak of such data risks revealing private data of End-Users
```